tests/integration/test_audio_io.py

```
@@ -65,4 +106,4 @@ def _pydub(path):
     )
 
     data, sr = audio_io.load_audio("f", target_sr=16000)
-    assert data.shape == (2, 2) or data.shape == (2,) and sr == 16000
+    assert data.shape == (2, 2) or data.shape == (2,) and sr == 16000
```

Ruff (0.14.8)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)